### ISSUE TITLE
Error installing

### ISSUE THREAD

**Issue by RabbitJackTrade** [dhvcc/rss-reader#1]
On trying to install (with either pip or pip3), I get the following error:

`ERROR: Command errored out with exit status 1:
     command: 'C:\Users\me\Anaconda3\python.exe' -c 'import sys, setuptools, tokenize; sys.argv[0] = '"'"'C:\\Users\\me\\AppData\\Local\\Temp\\pip-install-bua_52au\\rss-reader\\setup.py'"'"'; __file__='"'"'C:\\Users\\me\\AppData\\Local\\Temp\\pip-install-bua_52au\\rss-reader\\setup.py'"'"';f=getattr(tokenize, '"'"'open'"'"', open)(__file__);code=f.read().replace('"'"'\r\n'"'"', '"'"'\n'"'"');f.close();exec(compile(code, __file__, '"'"'exec'"'"'))' egg_info --egg-base 'C:\Users\me\AppData\Local\Temp\pip-pip-egg-info-rglk91eu'
         cwd: C:\Users\me\AppData\Local\Temp\pip-install-bua_52au\rss-reader\
    Complete output (9 lines):
    Traceback (most recent call last):
      File "<string>", line 1, in <module>
      File "C:\Users\me\AppData\Local\Temp\pip-install-bua_52au\rss-reader\setup.py", line 18, in <module>
        long_description=read('README.rst'),
      File "C:\Users\me\AppData\Local\Temp\pip-install-bua_52au\rss-reader\setup.py", line 7, in read
        return open(os.path.join(os.path.dirname(__file__), fname)).read()
      File "C:\Users\me\Anaconda3\lib\encodings\cp1252.py", line 23, in decode
        return codecs.charmap_decode(input,self.errors,decoding_table)[0]
    UnicodeDecodeError: 'charmap' codec can't decode byte 0x9d in position 183: character maps to <undefined>`

Any idea?

**Comment by dhvcc** [comment]
It seems like the problem here is the open() function of setup.py
Try replacing setup.py's read function with this
```python
def read(fname):
    return open(os.path.join(os.path.dirname(__file__), fname), encoding="utf8").read()
```

**Comment by dhvcc** [comment]
Also the project is kind of discontinued, but i think about refactoring it :)

**Comment by dhvcc** [comment]
The issue should be fixed in v3.0
